Source/JavaScriptCore/wasm/WasmSectionParser.cpp

```diff
@@ -113,11 +114,13 @@ auto SectionParser::parseType() -> PartialResult
                     RefPtr<TypeDefinition> group = TypeInformation::typeDefinitionForRecursionGroup(types);
                     RefPtr<TypeDefinition> projection = TypeInformation::typeDefinitionForProjection(group->index(), 0);
                     TypeInformation::registerCanonicalRTTForType(projection->index());
+                    m_info->rtts.uncheckedAppend(TypeInformation::getCanonicalRTT(projection->index()));
```

Comment:
Does this need a lock?

Reply:
Unnecessary since section-parser is single-thread. (FunctionParser is concurrent)